connect_ext_test.go

```diff
@@ -575,6 +575,40 @@ func Test_CustomCompression(t *testing.T) {
 	assert.Equal(t, response.Msg, &pingv1.PingResponse{Text: request.Text})
 }
 
+func Test_ConnectTimeout(t *testing.T) {
```

Comment:
Just noticed, how come some test names are `Test_XYZ` and others `TestXYZ` .. maybe let's choose one for consistency? (without the underscore, unless that adds something?)